routes/roomparti-routes.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../models");
+
+module.exports = function(app) {
+    app.get("/api/roomparticipants/", function(req,res){
+        db.RoomParticipants.findAll({})
+            .then(function(dbRoomParticipants) {
+            res.json(dbRoomParticipants);
+        });
+    });
+    app.post("/api/roomparticipants/", function(req, res) {
+        console.log(req.body);
+        db.RoomParticipants.create(req.body).then(function(dbRoomParticipants) {
```

Comment:
Be careful passing req.body to other functions or libraries, if the api changes in the future then req.body could have stuff on it you didn't intend. Should pull the properties off req.body you want and pass those.

Reply:
e.g:
const roomParticipant = {
  roomId: req.body.roomId,
  userId: req.body.roomId
  ...
};
db.roomParticipants.create(roomParticipant)
This is also a good place that you could validate that the information was valid too, if you wanted to do any sort of additional checking on top of what sequelize will do for you